### ISSUE TITLE
conflict tooltip css with "select2"

### ISSUE THREAD

**Issue by thearabbit** [Flyer53/jsPanel3#34]
conflict tooltip css with "select2"

![image](https://cloud.githubusercontent.com/assets/4495160/21838760/805ac854-d806-11e6-9d3a-2937b7b1184a.png)

Please help me.

**Comment by Flyer53** [comment]
Hi again,

You need to provide some more information ...

What is this image actually showing?
Is it just a jsPanel tooltip that's not positioned as you'd like it?
What is actually wrong and how would you like it to be?
What does your code look like?
What kind of css conflict is it with select2? 

It's the same as with your other help requests. You need to provide the necessary information!

**Comment by thearabbit** [comment]
I use `select2` for this select box.
When mouse over the select box, it show the current value (big).
And then I try to remove `jspanel3`, it is show normally.

**Comment by Flyer53** [comment]
Since you're obviously not able you describe your problem in a way I understand I created a simple example using select2 and a jsPanel tooltip. The only issue I saw is that the background color for the connector is not applied, so I corrected this with the callback function.

http://jspanel.de/media/demos/select2.html

Otherwise it works as expected.
If this still doesn't help, you definitely need to provide your code ...
